### DISCUSSION TITLE
加载离线数据很卡

### DISCUSSION THREAD

**Discussion by XiaoChu408** [zouyaoji/vue-cesium#550] [General]
我使用vc-prmitive-tileset直接加载本地的tileset.json，很卡。单个bd3m文件不大于15M，总共1G的数据。想问下开发者你们的vc-prmitive-tileset的数据大小是怎样的，要压缩成什么大小才能快速加载出来。

**Comment by zouyaoji** [comment]
比较吃显存，得上独显。

可以把url发来我测测看看。不行的话可用Cesium的沙盒和vue-cesium的codepen加你数据试试。
